iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/GeneratorImpl.java

```diff
@@ -247,12 +247,6 @@ public <R extends ReactiveLimits> R getReactiveLimits(Class<R> type) {
         return reactiveLimits.getReactiveLimits(type);
     }
 
-    @Override
-    public void remove() {
-        regulatingPoint.remove();
```

Comment:
shouldn't we keep the remove to unregister the regulatingPoint from the regulatingTerminal?

Reply:
Yes, this is a regression. Fixed.

Reply:
It's also the case for the SVC, VSC, Phase/RatioTapChanger, ShuntCompensator